api/action_radio.go

```diff
@@ -49,11 +78,28 @@ func (ar *ActionRadio) Start() error {
 	if ar.messageBatcher != nil {
 		return ar.messageBatcher.Start()
 	}
+	if ar.tickInterval > 0 {
+		go func() {
+			ticker := time.NewTicker(ar.tickInterval)
+			defer ticker.Stop()
+			for {
+				select {
+				case <-ticker.C:
+					ar.mutex.Lock()
+					ar.autoRadio()
+					ar.mutex.Unlock()
+				case <-ar.quit:
+					break
+				}
+			}
+		}()
+	}
```

Comment:
if we are able to remove the mutex, this can be replaced by existing `RecurringTask`?